playground/dune-project

```diff
@@ -33,15 +31,55 @@
   "This repository contains the playground served on https://ocaml.org/play.")
  (allow_empty)
  (depends
-  (ocaml
-   (>= 5.0.0))
-  (brr
-   (>= 0.0.6))
-  esbuild
-  (js_of_ocaml
-   (>= 5.0))
-  js-top-worker
-  code-mirror
-  merlin-js
-  rpclib
-  rpclib-lwt))
+  (astring (= 0.8.5))
+  (base-bigarray (= base))
+  (base-bytes (= base))
```

Comment:
I don't think it is customary to depend on all these `base-*` packages. After all, they're just transitive dependencies of the compiler selected.